cfme/tests/automate/custom_button/test_buttons.py

```diff
@@ -197,3 +197,41 @@ def test_button_avp_displayed(appliance, dialog, request):
         assert view.advanced.attribute(n).key.is_displayed
         assert view.advanced.attribute(n).value.is_displayed
     view.cancel_button.click()
+
+
+@pytest.mark.tier(3)
+def test_open_url_availability(appliance):
+    """Test open URL option should only available for Single display.
+
+    Prerequisities:
+        * Button Group
+
+    Steps:
+        * Create a Button with other than Single display options
+        * Assert flash message.
+    """
+
+    unassigned_gp = appliance.collections.button_groups.instantiate(
+        text="[Unassigned Buttons]", hover="Unassigned buttons", type="Provider"
+    )
+    button_coll = appliance.collections.buttons
+    button_coll.group = unassigned_gp  # Need for supporting navigation
+
+    view = navigate_to(button_coll, "Add")
+    view.fill(
+        {
+            "options": {
+                "text": "test_open_url",
+                "hover": "Open Url Test",
+                "image": "fa-user",
+                "open_url": True,
+            },
+            "advanced": {"system": "Request", "request": "InspectMe"},
+        }
+    )
+
+    for display in ["List", "Single and list"]:
```

Comment:
Consider parametrizing on these values, I believe its a common parametrization in these modules otherwise.